### ISSUE TITLE
Add new Wordpress user and other WP items not working correctly

### ISSUE THREAD

**Issue by equinoxevents** [mdjmplugin/mobile-dj-manager#87]
I am running WP 4.5.2 on a fresh install to test the viability of the MDJM platform for our new business as it seems feature rich.

We have run into a number of issues within the WP Admin area that are only apparent with the MDJM plugin active and all other plugins deactivated.

When adding a new WP user in the WP admin console, clicking on the 'show password' button has no effect. Even if all relevant fields are completed for a new user, it does not work stating that a password must be entered. Therefore no users can be added via WP. 

Other issues include not being able to move the information tiles in the dashboard area and npot being able to select some of the option tabs or the settings of some of the installed plugins such as 'launchpad'.

It should be noted that the issues disappear if the MDJM plugin is deactivated.

I have already created an MDJM user as an admin within WP and can provide the site url and pw via PM.

**Comment by mikeyhoward1977** [comment]
Do the issues disappear if the launchpad (or all other plugins) are de-activated? I have just tested on Safari and Firefox and in both instances the moving of the dashboard widgets and the show password option function normally.

Which browser are you using?

**Comment by mikeyhoward1977** [comment]
OK I investigated further and the issue only occurs when the launchpad plugin is installed. Oddly, even with launchpad de-activated.

As soon as I removed launchpad, all functionality was restored. There is obviously a conflict of some sort but it does not necessarily mean it's an MDJM issue.

I'll investigate further but I'd also suggest you contact support for Launchpad

**Comment by mikeyhoward1977** [comment]
@equinoxevents any further update from your side on this? Not been able to find any issue at this end and not other user reports of such an issue.

Will close this issue if I do not hear back over the next few days

**Comment by mikeyhoward1977** [comment]
Closing issue. No additional reports from any users, and no response from original poster

**Comment by equinoxevents** [comment]
Sorry for the delay in replying as I was abroad. 

Investigated further and uninstalled all plugins but the issues persisted. Luckily I have the luxury of being able to delete the whole WP install including the DB which I did then re-installed from scratch.

I uninstalled all of the default plugins and only installed the MDJM plugin

I can report that MDJM is currently woyrking without issues and over the next few days I will add the plugins I wanted (backing up before each install). I will leave out Launchpad as it is  not required at the moment.
